RoddyCore/src/de/dkfz/roddy/core/ExecutionContextReaderAndWriter.groovy

```diff
@@ -103,15 +103,15 @@ class ExecutionContextReaderAndWriter {
             for (String jobID : unknownJobs.keySet()) {
                 BEJob job = unknownJobs[jobID]
                 job.setJobState(map[job])
-                Roddy.getJobManager().addJobStatusChangeListener(job)
+                Roddy.getJobManager().addToListOfStartedJobs(job)
```

Comment:
Now I think, that addToListOfKnownJobs might be a much more suitable name. not all jobs added to the list are started. Some might just be read out from storage.

Reply:
Still... addToListOfTemporarilySurveilledJobs ?